libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/TwilioAdapter.cs

```diff
@@ -0,0 +1,243 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+using Twilio.Security;
+
+namespace Microsoft.Bot.Builder.Adapters.Twilio
+{
+    public class TwilioAdapter : BotAdapter
+    {
+        private readonly ITwilioAdapterOptions _options;
+
+        public TwilioAdapter(ITwilioAdapterOptions options)
+        {
+            _options = options;
+
+            if (options.TwilioNumber.Equals(string.Empty))
```

Comment:
Use string.IsNullOrEmpty, for example:
if (string.IsNullOrEmpty(options.TwilioNumber))

Reply:
Feedback applied. Thanks!